packages/@scope/jsii-calc-base/test/assembly.jsii

```diff
@@ -150,6 +150,6 @@
       "name": "IBaseInterface"
     }
   },
-  "version": "1.1.0",
-  "fingerprint": "piaiqNxSj1XkycnXHPRG4697t6DpKGgkya6effAlXKI="
+  "version": "0.0.0",
```

Comment:
Oh cool we no longer need to update tests in version bumps!

Reply:
yeah, this does add some complexity to how we build though.

Since tests rely on version numbers not changing, we have to build+test before we run align-version in our build pipeline. Then after running align-version, I'm wary of running package.sh without running build again because if for some reason we changed how align-version works or otherwise do something that could affect the build, then I'd like to know about that failure.

Because of this I actually changed buildspec.yml to do build+test+align-version+build+pack. This seems to be the safest thing for now? @eladb am I being irrational by worrying about this?

Reply:
Yes that sounds like what I would do. Not ideal but will work I believe until we figure out some other way...